src/pudl/metadata/resources/eia860.py

```diff
@@ -240,6 +240,8 @@
                     "heat_rate_by_generator_monthly",
                     "mcoe_monthly",
                     "mcoe_generators_monthly",
+                    "ownership_eia860",
+                    "denorm_ownership_eia860",
```

Comment:
ha i just asked the same thing for the plant fk relationship. it doesn't seem like it should be necessary anymore.

Reply:
No, it shouldn't need to be excluded any more, but I think we do need to **add** `owner_utility_id_eia` as one of the columns which is used to construct the automatic FK relationships. This would be done in the FK rules section of the utility entity and annual utility tables.

Reply:
See this section in the `utilities_eia860` resource definition below:
```py
            "foreign_key_rules": {
                "fields": [
                    ["utility_id_eia", "report_date"],
                    # Failing because this column is not harvested in the old
                    # system. TODO: re-enable when we switch to new system.
                    # https://github.com/catalyst-cooperative/pudl/issues/1196
                    # ["owner_utility_id_eia", "report_date"],
```